distro.py

```diff
@@ -1034,6 +1041,8 @@ def _parse_distro_release_content(line):
                 distro_info['version_id'] = m.group(2)[::-1]
             if m.group(1):
                 distro_info['codename'] = m.group(1)[::-1]
+        elif line:
+            distro_info['name'] = line.strip()
```

Comment:
the content of [manjaro-release](https://github.com/nir0s/distro/pull/130/files/94ef753bc172803d4d981be6532ae39f03327e13#diff-c70a88ce1c8376af239b0b2083d1d3c3) isn't matched before.